Sources/QRCodeModel.swift

```diff
@@ -30,10 +30,12 @@ struct QRCodeModel {
     private let encodedText: QR8bitByte
     private var dataCache: [Int]
     
-    init?(text: String, typeNumber: Int, errorCorrectLevel: QRErrorCorrectLevel) {
-        guard let encoded = QR8bitByte(text) else {
+    init?(text: String, typeNumber: Int, errorCorrectLevel: QRErrorCorrectLevel, textEncoding: String.Encoding = .utf8) {
```

Comment:
Since `textEncoding` is closely related to `text`, can we make it the 2nd argument and change the name to just `encoding`?

Reply:
I agree, However I made it like this because from what I remember sometimes the parameters order can affect existing code.
I believe that this way it won't affect existing users.

If you think otherwise it's fine by me, since Swift use named parameters anyways.

Reply:
This parameter is entirely additional and provides default value, so where the position is won't affect compatibility with previous interfaces.